### PR TITLE
CIP-0026 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #671: CIP-0026 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #672.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #671.

@KtorZ @michaelpj @polinavino this is such a well documented, apparently broadly useful proposal... were there really never any implementations of it besides [offchain-metadata-tools](https://github.com/input-output-hk/offchain-metadata-tools)?  Could there in turn have been anything that included `offchain-metadata-tools` to provide some commercial or community implementation?

([updated proposal](https://github.com/rphair/CIPs/blob/remediate-0026/CIP-0026/README.md))

### CODE REVIEW

**Review by michaelpj — APPROVED** [review]
Ultimately, I think nobody was really willing to put will behind an off-chain solution :shrug:

**Comment by Ryun1** [comment]
The [cardano-token-registry](https://github.com/cardano-foundation/cardano-token-registry) is probably the most significant implementation :)

File: CIP-0026/README.md [hunk 1]
```diff
@@ -463,23 +469,29 @@ Besides, on-chain storage comes with considerable downsides:
...
+### Backwards compatibility
 
 See [Special Case: Phase-1 Monetary Policies](#special-case--phase--1-monetary-policies) which covers existing implementations. 
 
-# Reference implementation
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] Document commercial or community implementations of any of the use cases described above.
```
Comment (Ryun1) [inline CIP-0026/README.md:486]:
```suggestion
- [x] Document commercial or community implementations of any of the use cases described above.
  - [cardano-token-registry](https://github.com/cardano-foundation/cardano-token-registry)
```

File: CIP-0026/README.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
 ---
 CIP: 26
 Title: Cardano Off-Chain Metadata
-Authors: Matthias Benkort <matthias.benkort@iohk.io>, Michael Peyton Jones <michael.peyton-jones@iohk.io>, Polina Vinogradova <polina.vinogradova@iohk.io>
-Comments-Summary: No comments yet
-Comments-URI: https://github.com/CardanoFoundation/CIPs/pulls/112
-Status: Draft
-Type: Process
+Status: Proposed
```
Comment (Ryun1) [inline CIP-0026/README.md:4]:
```suggestion
Status: Active
```
Since the token registry

File: CIP-0026/README.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+Category: Metadata
+Authors:
+  - Matthias Benkort <matthias.benkort@iohk.io>
+  - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+  - Polina Vinogradova <polina.vinogradova@iohk.io>
+Implementors: []
```
Comment (Ryun1) [inline CIP-0026/README.md:10]:
```suggestion
Implementors:
  - cardano-token-registry <https://github.com/cardano-foundation/cardano-token-registry>
```

**Review by Ryun1 — APPROVED** [review]
LGTM

just adding a implementation of [cardano-token-registry](https://github.com/cardano-foundation/cardano-token-registry)

**Review by KtorZ — APPROVED** [review]

**Pull request merged**